### PR TITLE
executor: "fetchShowTableStatus" should append an integer to the third column instead of a string

### LINKED ISSUE #5605: show tables status failed with panic
## 1. What did you do?

```
create table a0(a bigint);
create table a1(a bigint);
create table a2(a bigint);
create table a3(a bigint);
create table a4(a bigint);
create table a5(a bigint);
create table a6(a bigint);
create table a7(a bigint);
create table a8(a bigint);
create table a9(a bigint);
create table b0(a bigint);
create table b1(a bigint);
create table b2(a bigint);
create table b3(a bigint);
create table b4(a bigint);
create table b5(a bigint);
create table b6(a bigint);
create table b7(a bigint);
create table b8(a bigint);
create table b9(a bigint);
create table c0(a bigint);
create table c1(a bigint);
create table c2(a bigint);
create table c3(a bigint);
create table c4(a bigint);
create table c5(a bigint);
create table c6(a bigint);
create table c7(a bigint);
create table c8(a bigint);
create table c9(a bigint);
create table d0(a bigint);
create table d1(a bigint);
create table d2(a bigint);
create table d3(a bigint);
create table d4(a bigint);
create table d5(a bigint);
create table d6(a bigint);
create table d7(a bigint);
create table d8(a bigint);
create table d9(a bigint);
```

## 3. What did you see instead?

```sql
TiDB(localhost) > show table status;
ERROR 2013 (HY000): Lost connection to MySQL server during query
```

from tidb log:
```txt
2018/01/10 14:30:33.641 conn.go:410: [error] lastCmd show table status, runtime error: slice bounds out of range, goroutine 1510 [running]:
github.com/pingcap/tidb/server.(*clientConn).Run.func1(0xc4214aa750, 0xc4218b5e0f)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:408 +0x10e
panic(0x1cd1e60, 0x285a490)
        /Users/jianzhang.zj/opt/go-1.9/src/runtime/panic.go:491 +0x283
github.com/pingcap/tidb/util/chunk.(*Chunk).Append(0xc4210c1ca0, 0xc4210c1ce0, 0x0, 0x29)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/util/chunk/chunk.go:186 +0x771
github.com/pingcap/tidb/executor.(*ShowExec).NextChunk(0xc4213804b0, 0x2e92808, 0xc4216ae440, 0xc4210c1ca0, 0x12, 0x12)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/executor/show.go:112 +0xad
github.com/pingcap/tidb/executor.(*recordSet).NextChunk(0xc421591a40, 0x2e92808, 0xc4216ae440, 0xc4210c1ca0, 0xc421591a40, 0xc4210c1ca0)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/executor/adapter.go:103 +0x56
github.com/pingcap/tidb/server.(*tidbResultSet).NextChunk(0xc42108c930, 0x2e92808, 0xc4216ae440, 0xc4210c1ca0, 0x4, 0x400)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/driver_tidb.go:302 +0x51
github.com/pingcap/tidb/server.(*clientConn).writeChunks(0xc4214aa750, 0x2e92808, 0xc4216ae440, 0x264ba60, 0xc42108c930, 0x0, 0xc42108c930, 0x1)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:936 +0x3ef
github.com/pingcap/tidb/server.(*clientConn).writeResultset(0xc4214aa750, 0x2e92808, 0xc4216ae440, 0x264ba60, 0xc42108c930, 0xc4202e0000, 0x0, 0x0)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:861 +0x668
github.com/pingcap/tidb/server.(*clientConn).handleQuery(0xc4214aa750, 0x2e92808, 0xc4216ae440, 0xc420280ec1, 0x11, 0x0, 0x0)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:810 +0x115
github.com/pingcap/tidb/server.(*clientConn).dispatch(0xc4214aa750, 0xc420280ec1, 0x12, 0x12, 0x0, 0x0)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:592 +0x595
github.com/pingcap/tidb/server.(*clientConn).Run(0xc4214aa750)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/conn.go:452 +0x1b8
github.com/pingcap/tidb/server.(*Server).onConn(0xc42037ea80, 0x264fa20, 0xc4217cc9c8)
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/server.go:316 +0x144
created by github.com/pingcap/tidb/server.(*Server).Run
        /Users/jianzhang.zj/opt/gopath/src/github.com/pingcap/tidb/server/server.go:262 +0x496
```

## 4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
```shell
$./bin/tidb-server -V
Release Version: v1.1.0-alpha-401-gb7e7cd5d9-dirty
Git Commit Hash: b7e7cd5d9a0c38fe248e45bf1e89fe1e3f904aed
Git Commit Branch: master
UTC Build Time:  2018-01-10 06:02:25
```

### PR DESCRIPTION
**Pull request by zz-jason**
fix #5605

### CODE REVIEW

**Comment by zz-jason** [comment]
@XuHuaiyu @shenli PTAL

**Comment by shenli** [comment]
LGTM

File: executor/show_test.go [hunk 1]
```diff
@@ -535,3 +535,53 @@ func (s *testSuite) TestCollation(c *C) {
...
+	tk.MustExec(`create table d3(a bigint);`)
+	tk.MustExec(`create table d4(a bigint);`)
+	tk.MustExec(`create table d5(a bigint);`)
+	tk.MustExec(`create table d6(a bigint);`)
+	tk.MustExec(`create table d7(a bigint);`)
+	tk.MustExec(`create table d8(a bigint);`)
+	tk.MustExec(`create table d9(a bigint);`)
+
+	// It's not easy to test the result contents because every time the test runs, "Create_time" changed.
+	tk.MustExec("show table status;")
```
Comment (coocood) [inline executor/show_test.go]: We can just create one table and check the third column's value.

**Pull request merged**